spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java

```diff
@@ -276,13 +286,21 @@ else if (messageListener instanceof MessageListener) {
 			throw new IllegalStateException("messageListener must be 'MessageListener' "
 					+ "or 'AcknowledgingMessageListener', not " + messageListener.getClass().getName());
 		}
-		if (getTaskExecutor() == null) {
-			setTaskExecutor(
-					new SimpleAsyncTaskExecutor(getBeanName() == null ? "kafka-" : (getBeanName() + "-kafka-")));
+		if (getConsumerTaskExecutor() == null) {
+			SimpleAsyncTaskExecutor pooledExecutor = new SimpleAsyncTaskExecutor(
```

Comment:
s/pooledExecutor/consumerTaskExecutor
sounds better for me. And with that name I'd even don't pay attention that it isn't `ThreadPoolTaskExecutor` :smile: